Documentation/Testing/ExtensionTesting.rst

```diff
@@ -42,8 +42,8 @@ About this chapter and what it does *not* cover, first.
   development of that package is closely bound to Core development and has a relatively high
   development speed. It does contain breaking patches per major Core versions, but it should
   not contain breaking patches for existing major Core branches. If you now set up testing
-  using `typo3/testing-framework` with TYPO3 Core version 9, it should not break within v9's
-  lifetime. But it is likely to break if you upgrade to version 10 or later and may need adaption
+  using `typo3/testing-framework` with TYPO3 Core version 11, it should not break within v11's
```

Comment:
The whole chapter needs a rewrite from someone who knows the testing framework. Now, the testing framework supports two Core versions in one branch, so v6 is for TYPO3 v10 and TYPO3 v11 and the upcoming v7 is for TYPO3 v11 and v12. But we can leave it for now.

Reply:
Yeah, it is on my todo list